staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/testing/testserver.go

```diff
@@ -83,13 +84,13 @@ func NewDefaultTestServerOptions() *TestServerInstanceOptions {
 // files that because Golang testing's call to os.Exit will not give a stop channel go routine
 // enough time to remove temporary files.
 func StartTestServer(t Logger, _ *TestServerInstanceOptions, customFlags []string, storageConfig *storagebackend.Config) (result TestServer, err error) {
-	stopCh := make(chan struct{})
+	ctx, cancel := context.WithCancelCause(context.Background())
```

Comment:
We cannot use ktesting here because it is not a staging repo (yet).

Reply:
Add TODO?

Reply:
Added.